README.md

```diff
@@ -1,27 +1,39 @@
 # ssb-db
 
-[secret-stack](https://github.com/ssbc/secret-stack) plugin which provides storing of valid secure-scuttlebutt messages in an append-only log.
+[secret-stack](https://github.com/ssbc/secret-stack) plugin which provides storing of valid secure-scuttlebutt 
+messages in an append-only log.
 
-## What does it do?
+## Table of contents
 
-`ssb-db` provides tools for dealing with unforgeable append-only message 
-feeds. You can create a feed, post messages to that feed, verify a feed created by
-someone else, stream messages to and from feeds, and more (see [API](#api)).
+[What does it do?](#what-does-it-do) | 
+[Example](#example) | 
+[Concepts](#concepts) | 
+[API](#api) | 
+[Stability](#stability) | 
+[License](#License) | 
+___
 
-"Unforgeable" means that only the owner of a feed can modify that feed, as
-enforced by digital signing (see [Security properties](#security-properties)).
-This property makes ssb-db useful for peer-to-peer applications. ssb-db also
-makes it easy to encrypt messages.
+# What does it do?
 
-## Example
+`ssb-db` provides tools for dealing with unforgeable append-only message feeds. You can create a feed, post 
+messages to that feed, verify a feed created by someone else, stream messages to and from feeds, and more 
+(see [API](#api)).
 
-In this example, we create a feed, post a signed message to it, then create a stream 
-that reads from the feed.
+*Unforgeable* means that only the owner of a feed can modify that feed, as enforced by digital signing 
+(see [Security properties](#security-properties)).
+
+This property makes `ssb-db` useful for peer-to-peer applications. `ssb-db` also makes it easy to encrypt 
+messages.
+
+# Example
+
+In this example, we create a feed, post a signed message to it, then create a stream that reads from the feed.
 
 ``` js
 /**
  * create an ssb-db instance and add a message to it.
  */
+var pull = require('pull-stream')
```

Comment:
Great addition. 
Docs can improve significantly by better incorporating and introducing pull-stream, given its significance to the stack